docs/flow/modin/base.rst

```diff
@@ -0,0 +1,12 @@
+BasePandasDataset
+"""""""""""""""""
+
+The class is base for Modin ``DataFrame`` and ``Series`` classes and implements functionality that is common to both
```

Comment:
I would clarify here that this is the pandas API.

Reply:
Ok I added mention of pandas API, but if you still don't like it, please simply write that you want to see here or add your own commit. It is not easy to guess from a brief comment.